userpatches/customize-image.sh

```diff
@@ -204,6 +204,10 @@ SetupOtherServices() {
 	copy_overlay /lib/systemd/system/iperf2.service -o root -g root -m 644
 	copy_overlay /lib/systemd/system/iperf2-udp.service -o root -g root -m 644
 
+        display_alert "Setup service" "networkinfo" "info"
+        copy_overlay /lib/systemd/system/networkinfo.service -o root -g root -m 644
+        systemctl enable networkinfo
```

Comment:
Please use tab instead of spaces to maintain the same pattern as the rest of the file.

Reply:
Fixed now